src/Game.java

```diff
@@ -0,0 +1,64 @@
+import java.util.*;
+
+public class Game {
+    private List<String> words = new ArrayList<>();
+    Random random = new Random();
+    WordsDatabase database = new WordsDatabase(words, random);
+    public final int exit = 0;
+    public final int play = 1;
+    private int opition = 1;
+    Scanner sc = new Scanner(System.in);
+
+    public void hangman() {
+        try {
+            while (opition != exit) {
+
+
+                switch (opition) {
+                    case exit:
+                        System.out.println("Koniec gry.");
+                        sc.close();
+                        break;
+                    case play:
+                        game();
+                        break;
+                    default:
+                        System.out.println("Nie ma takiej opcji.");
+                }
+                System.out.println("Dostępne opcję:");
+                System.out.println("0 - wyjście z programu.");
+                System.out.println("1 - graj dalej");
+                opition = sc.nextInt();
+                sc.nextLine();
+
+            }
+        } catch (InputMismatchException e) {
+            System.out.println("Nie mozna podawać znaków niezgodnych z poleceniem.");
```

Comment:
What happens if exception is thrown? Is user still allowed to play or does the game end?

Reply:
Game is ended, i did that because way that i know how to make game played isn't "pretty", would you like me to do it anyway ?

Reply:
I added boolen and while, now game is active even user will write sth wrong.